### DISCUSSION TITLE
Add "case "8UC4":" to decodeBGRA8 switch in decodeRawImage

### DISCUSSION THREAD

**Discussion by JordanHumphries** [foxglove/community#766] [Q&A]
Foxglove Studio claim it cannot decode 8UC4 but it supports it.

- Version: latest
- Platform: ubuntu
- Data source ( ROS1, Foxglove Bridge):

**Steps To Reproduce**
Stream an 8UC4 encoded image and using the Image Panel.

**Expected Behavior**
Render Image

**Comment by foxhubber** [comment]
Internal tracking ticket: [FG-4152](https://linear.app/foxglove/issue/FG-4152/add-case-8uc4-to-decodebgra8-switch-in-decoderawimage-[foxglovestudio)

**Comment by jtbandes** [comment]
I believe you are right that this would be a valuable/correct change. However, this is not a bug since the documentation and app behavior are not broken. Please be mindful of the issue template and guidelines when creating an issue:

<img width="581" alt="image" src="https://github.com/foxglove/studio/assets/14237/e72378af-00ba-4c38-bd85-d89994a86104">
